etl/steps/data/garden/un/2024-07-25/refugee_data.meta.yml

```diff
@@ -0,0 +1,279 @@
+  description_idp_return: |-
+    Returned [internally displaced people](#dod:internally-displaced) refers to those IDPs who were beneficiaries of UNHCR’s protection and assistance activities, and who returned to their areas of origin or habitual residence during the calendar year. In practice, however, operations may assist IDP returnees for longer periods.
+  description_naturalization: |-
+    Naturalized [refugees](#dod:refugee) refers to those refugees that have naturalized in their country of asylum. Naturalization is the legal act or process by which a non-citizen in a country may acquire citizenship or nationality of that country. It is used as a proxy measure of local integration. The International Recommendations on Refugee Statistics make several recommendations on appropriate indicators for measuring and quantifying local integration in a way that is comparable and consistent across different contexts. However, the availability of such data about refugees remains very poor. Even data on naturalization is limited by the uneven availability of data and poor coverage as well as policy and legal changes over time. In particular, it can be difficult to distinguish between the naturalization of refugees and non-refugees. Therefore, the data are only indicative at best and provide an underestimate of the extent to which refugees are naturalized.
+  description_resettlement: |-
+    Resettled [refugees](#dod:refugee) are those who have who have been resettled to another country. Resettlement is used to assist refugees in countries that cannot provide them with appropriate protection and support. Resettlement is primarily facilitated by UNHCR in most countries around the world, although significant private sponsorship schemes do exist as well (e.g. in Canada).
+
+  description_rolling_avg: |-
+    Some metrics, which are reported annually, come with high levels of volatility which limits their usability.
+    However, volatility and sudden changes are important signals for understanding migration due to conflict or disaster. To make these changes more understandable, while maintaining the important signals of year-to-year changes, we have re-calculated annual data as the rolling five-year average here.
+
+dataset:
+  update_period_days: 180
+
+
+tables:
+  refugee_data:
+    variables:
+      # populations by origin country
+      refugees_under_unhcrs_mandate_origin:
+        title: Refugees by country of origin
+        description_short: Number of [refugees](#dod:refugee) in all regions by the country of origin.  This measures the total number of refugees from the country; not the number per year.
```

Comment:
Extra space after the period.

Reply:
I think there should be a clear first definition of refugee in text, not in the DoD. The definition can maybe expanded in the DoD

Reply:
In the charts we currently have on the website we do not define refugees in the FASTT. I assumed the term is well enough understood that a DOD is sufficient, but I can change it if you disagree :)